### DISCUSSION TITLE
How to Access Stress Tensor in PxSoftBody Using eCOMPUTE_STRESS_TENSOR Flag?

### DISCUSSION THREAD

**Discussion by op585** [NVIDIA-Omniverse/PhysX#301] [Q&A]
![image](https://github.com/user-attachments/assets/d8813402-f27f-4d51-b5d9-54047db48fc0)

Hello,

I have been working with NVIDIA PhysX for soft body simulations and came across the eCOMPUTE_STRESS_TENSOR flag in the PxSoftBodyFlag structure. According to the documentation (see attached image), this flag enables the computation of a Cauchy stress tensor for every tetrahedron in the simulation mesh, and these tensors can be accessed through the soft body direct API.

I have successfully set the eCOMPUTE_STRESS_TENSOR flag to true, but I am having trouble figuring out how to access the stress tensors. The documentation does not provide clear instructions on the exact API calls or methods needed to retrieve these tensors.

Could anyone provide guidance or examples on how to access the stress tensors after enabling the eCOMPUTE_STRESS_TENSOR flag?

Thank you for your assistance!

**Comment by heiyun717** [comment]
I have encountered the same problem. How did you solve it later
